extensions/libraries/redcore/api/payment/document/document.php

```diff
@@ -96,6 +96,7 @@ public function render($cache = false, $params = array())
 
 		// Get the payment string from the buffer.
 		$content = $this->getBuffer();
+		$jsonEncodedContent = json_encode($content, JSON_UNESCAPED_SLASHES);
```

Comment:
why is it forced to json now?
maybe you should check $this->_mime format parameter and render if differently if there is a json format or txt